onnx/external_data_helper.py

```diff
@@ -99,11 +101,11 @@ def set_external_data(tensor,  # type: TensorProto
             entry.value = str(v)
 
 
-def convert_model_to_external_data(model, all_tensors_to_one_file=True, location=None, size_threshold=1024):
-    # type: (ModelProto, bool, Optional[Text], int) -> None
+def convert_model_to_external_data(model, all_tensors_to_one_file=True, location=None, size_threshold=1024, convert_attribute=False, model_file_path=None):
```

Comment:
I set `convert_attribute` to False, this will change the behavior of the end-user. But, I didn't think we wanted to set it to True

Reply:
After speaking with Ashwini, we decided that setting the `convert_attribute` value to false is okay even if that will change the current behavior as converting attribute was not an intended behavior in the first place